File: features/promo/gamification/src/main/java/com/tokopedia/gamification/data/entity/TokoPointDetailEntity.java
package com.tokopedia.gamification.data.entity;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class TokoPointDetailEntity {

    @Expose
    @SerializedName("tokopoints")
    TokoPointEntity tokoPoints;

    public TokoPointEntity getTokoPoints() {
        return tokoPoints;
    }

    public void setTokoPoints(TokoPointEntity tokoPoints) {
        this.tokoPoints = tokoPoints;
    }

    @Override
    public String toString() {
        return "TokoPointDetailEntity{" +
                "tokoPoints=" + tokoPoints +
                '}';
    }
}

File: features/promo/tokopoints/src/main/java/com/tokopedia/tokopoints/view/model/TokoPointDetailEntity.kt
package com.tokopedia.tokopoints.view.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class TokoPointDetailEntity(
    @Expose
    @SerializedName("tokopoints")
    var tokoPoints: TokoPointEntity = TokoPointEntity()
)
